README.md

```diff
@@ -41,7 +41,8 @@ This tap:
         "password": "your-jira-password",
         "base_url": "https://your-jira-domain",
         "user_agent": "<user-agent>",
-        "request_timeout": 300
+        "request_timeout": 300,
+        "groups": "jira-administrators,site-admins,jira-software-users"
```

Comment:
```suggestion
        "groups": "jira-administrators, site-admins, jira-software-users"
```

Reply:
As shown in the [code,](https://github.com/singer-io/tap-jira/blob/TDL-19198-Provide-description-of-groups-in-README/tap_jira/streams.py#L228) when we split the 'groups' value, we are not removing extra spaces. So if we are giving 'groups' value with space as shown in the above suggestion, we are getting a list `["jira-administrators", " site-admins", " jira-software-users"]` instead of `["jira-administrators", "site-admins", "jira-software-users"]`.

Reply:
I think extra space should not fail the tap. Ideally after a split we should strip() the leading spaces.

Reply:
In the existing code, we are not using strip, but this suggestion is right, we should ignore extra spaces, so added strip() and also update README as per code change.